lnrpc/walletrpc/config_active.go

```diff
@@ -44,7 +44,7 @@ type Config struct {
 
 	// Wallet is the primary wallet that the WalletKit will use to proxy
 	// any relevant requests to.
-	Wallet lnwallet.WalletController
+	Wallet *lnwallet.LightningWallet
```

Comment:
Why this change? I think the interface needs to be left here so it's easier to write unit tests. Can we add the method `CurrentNumAnchorChans` to that interface and use it here?

Reply:
I think the `WalletController` interface is for non Lightning Network specifications, not sure if it be fine to add  `CurrentNumAnchorChans` there.